templates/questions/security.yaml

```diff
@@ -5,15 +5,23 @@
       type: dict
       additional_attrs: true
       attrs:
-        - variable: PUID
-          label: "Process User ID - PUID"
-          description: "When supported by the container, this sets the User ID running the Application Process. Not supported by all Apps"
+        - variable: editsecurity
+          label: "Enable Security and Permission Settings"
+          description: "By enabling this you override predefined set values."
```

Comment:
This getting confusing, I wouldn't even know what this means and I made this.

Reply:
It's important to note that people should know what is behind a button BEFORE clicking the button.
PUID and PGID has no meaning within kubernetes (and shouldn't either)

So it definately need MUCH better naming.